client-src/elements/chromedash-app.js

```diff
@@ -253,7 +274,8 @@ class ChromedashApp extends LitElement {
                 .user=${this.user}
                 .appTitle=${this.appTitle}
                 .googleSignInClientId=${this.googleSignInClientId}
-                .currentPage=${this.currentPage}>
+                .currentPage=${this.currentPage}
+                ?isNewFeatures=${true}>
```

Comment:
Is this supposed to always be true?
I thought that maybe you were trying to offer the user two modes with a boolean to determine which mode was active.  But, there needs to be some way for the user to affect that boolean's value.

Reply:
I actually had a bit trouble setting the correct boolean values because for /features, the chromedash-header is rendered in _base.html, while it is here for /newfeatures. We have two ways to render the same tabs. Perhaps we can talk more in our meeting

Reply:
Per our offline discussions, we will stick to static tab for now.